.npmignore

```diff
@@ -0,0 +1,26 @@
+# https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package
+
+# build tool config
+Jenkinsfile
+.travis.yml
+
+# code examples
+examples
+
+# docs
+CONTRIBUTING.md
+
```

Comment:
The npm docs say CHANGELOG (and its variants) are always included. Not sure why.
Even if I add it to .npmignore it still appears in the package.